.env.development

```diff
@@ -4,4 +4,6 @@ export DB_HOST=127.0.0.1
 export DB_NAME=xrt_dev
 export DB_USER=postgres
 export DB_PASSWORD=postgres
-export ADMIN_PASSWORD=admin
+export DASHBOARD_USER=patata
```

Comment:
those env vars are not used in development. Can you use it in dev.exs?

Reply:
oh that's true, it's leftovers from testing, I'll remove them